pyvo/dal/adhoc.py

```diff
@@ -18,7 +18,7 @@
 from astropy.units import Quantity, Unit
 from astropy.units import spectral as spectral_equivalencies
 
-from astropy.io.votable.tree import Resource, Group
+from astropy.io.votable.tree import Resource, Group, VOTableFile, TableElement
```

Comment:
Ah, this is real, `TableElement` is not available in the oldest supported astropy, you need to add a try/exept like this:

```suggestion
from astropy.io.votable.tree import Resource, Group, VOTableFile

try:
    # Workaround astropy deprecation, remove try/except once >=6.0 is required
    from astropy.io.votable.tree import TableElement
except ImportError:
    from astropy.io.votable.tree import Table as TableElement
```

Reply:
I've noticed that but didn't bother with it until I get confirmation that this is an acceptable solution. It's still a draft...